sdk/maps/azure-maps-route/src/test/resources/getroutedirections.json

```diff
@@ -0,0 +1,241 @@
+{
+    "formatVersion": "0.0.12",
+    "routes": [
+        {
+            "summary": {
+                "lengthInMeters": 1147,
+                "travelTimeInSeconds": 128,
+                "trafficDelayInSeconds": 0,
+                "trafficLengthInMeters": 0,
+                "departureTime": "2022-03-15T02:24:24+01:00",
+                "arrivalTime": "2022-03-15T02:26:31+01:00"
+            },
+            "legs": [
+                {
+                    "summary": {
+                        "lengthInMeters": 1147,
+                        "travelTimeInSeconds": 128,
+                        "trafficDelayInSeconds": 0,
+                        "trafficLengthInMeters": 0,
+                        "departureTime": "2022-03-15T02:24:24+01:00",
+                        "arrivalTime": "2022-03-15T02:26:31+01:00"
+                    },
+                    "points": [
+                        {
+                            "latitude": 52.5093,
```

Comment:
Again, from reading the code it seems like toRouteQueryString will return "lon,lat:lon,lat", which should not match this result here.